runtime/src/accounts.rs

```diff
@@ -288,7 +288,7 @@ impl Accounts {
                             (account_override.clone(), 0)
                         } else {
                             self.accounts_db
-                                .load_with_fixed_root(ancestors, key)
+                                .load_with_fixed_root(ancestors, key, true)
```

Comment:
If `!message.is_writable(i)` then `remove_from_read_only_cache` could be `false`, right?

Reply:
yes. I was hoping @alessandrod would fix that up when he tried this out

Reply:
Yep I've been trying with this change